src/scripts/cancel_older_approvals.py

```diff
@@ -0,0 +1,49 @@
+import os
+import requests
+
+circleci_api_token = os.getenv('CIRCLECI_TOKEN')
+CIRCLE_WORKFLOW_ID = os.getenv('CIRCLE_WORKFLOW_ID')
+CIRCLE_PROJECT_USERNAME = os.getenv('CIRCLE_PROJECT_USERNAME')
+VCS="gh"
+CIRCLE_PROJECT_REPONAME = os.getenv('CIRCLE_PROJECT_REPONAME')
+BASE_URL="https://circleci.com/api/v2"
+TOKEN=f'circle-token={circleci_api_token}'
+
+current_wf_url = f'{BASE_URL}/workflow/{CIRCLE_WORKFLOW_ID}?{TOKEN}'
+
+
+try:
+    print(current_wf_url)
```

Comment:
Ouch!  We don't want to be printing the token out.   Remove it and rotate the token (assume it's already leaked in CI logs).

Can you also migrate this script to using the newer header auth format?   You already do this in `check_parallel.py`:
```
Circle-Token: ${CIRCLE_TOKEN}
```